students/jemielity_kamil/lesson_02_flow_control/README.md

```diff
@@ -0,0 +1,11 @@
+### Lesson 2 - Flow control
```

Comment:
Please remove all unnecessary files from this PR like files from lesson 02 and 03